src/lib/grid/grid.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../theme/variables.scss';
```

Comment:
I'd like to see a 1:1 relationship between `scss` files and their components.

Reply:
Good call. I made the change.